packages/netlify-cms-backend-github/package.json

```diff
@@ -17,7 +17,7 @@
   "scripts": {
     "develop": "yarn build:esm --watch",
     "build": "cross-env NODE_ENV=production webpack",
-    "build:esm": "cross-env NODE_ENV=esm babel src --out-dir dist/esm --ignore \"**/__tests__\" --root-mode upward",
+    "build:esm": "cross-env NODE_ENV=esm babel src --out-dir dist/esm --ignore \"**/__tests__\" --root-mode upward --extensions \".js,.jsx,.ts,.tsx\" --source-maps inline",
```

Comment:
I'm a bit worried about bloating our esm files as the maps tend to be a few times larger than the source they represent. Did you find it to not have a big impact on our output size?